packages/contracts-sdk/src/lib/contracts-sdk.ts

```diff
@@ -1361,7 +1378,7 @@ https://developer.litprotocol.com/v3/sdk/wallets/auth-methods/#auth-method-scope
     try {
       const res = await this.rateLimitNftContract.write.mint(expiresAt, {
         value: mintCost,
-        gasLimit: gasLimit || GAS_LIMIT,
+        gasLimit: gasLimit,
```

Comment:
Should this call also be using our new `estimateGasWithMargin()`? Do we not want dynamic gas limit calculation on every call? If we're intentionally not using that logic here, a comment would be a good idea describing _why_ we explicitly left this as-is

Reply:
I added the margin to all calls as @Ansonhkg suggested
In particular does not seem to be a bad thing as any excess gas will be returned